### ISSUE TITLE
product_uuid not present on paravirtual EC2 VMs

### ISSUE THREAD

**Issue by rade** [weaveworks/weave#2021]
I created a VM off a stock ubuntu 15.10 AMI (I think it was https://console.aws.amazon.com/ec2/home?region=eu-west-1#launchAmi=ami-f5952a86, located via https://cloud-images.ubuntu.com/locator/ec2/), installed some stuff in it, snapshotted it and created an image from that.

In VMs created from that image...

```
root@ip-172-31-1-14:~# cat /sys/class/dmi/id/product_uuid
cat: /sys/class/dmi/id/product_uuid: No such file or directory
root@ip-172-31-1-14:~# cat /sys/hypervisor/uuid
ec2f2c13-5ec7-fa7f-d174-626fce48d68f
```
(https://github.com/weaveworks/weave/pull/1866#issuecomment-171365527 led me to the latter)

**Comment by rade** [comment]
I've just tried to retrace my steps...

Launching a VM from the above AMI does bring up an instance where product_uuid exists. and cloning it does too.

Then I took a closer look at the images that I had created earlier. Turned out they were of a different type (t1.micro instead of t2.micro) and used "paravirtual" virtualisation hypervisor instead of "hvm".

That makes all the difference. Using https://console.aws.amazon.com/ec2/home?region=eu-west-1#launchAmi=ami-54972827, and choosing a t1.micro, does create a VM exhibiting the symptoms.
